indico/modules/events/abstracts/templates/abstract/judge.html

```diff
@@ -37,43 +37,97 @@
                             'negative': _("No reviewer proposed to accept this abstract"),
                             'mixed': _("Not all reviewers proposed to accept this abstract")} %}
     {% set state = abstract.get_track_reviewing_state(track) %}
-    <span class="i-tag outline {{ mapping_class[state.name] }}" title="{{ mapping_title[state.name] }}">
-        {{ state.title -}}
-    </span>
+    <div class="state-icon {{ mapping_class[state.name] }} flexrow f-a-center f-j-center"
+         title="{{ mapping_title[state.name] }}">
+        <span
+            {%- if state.name in ['positive', 'conflicting'] %}
+                class="icon-checkmark"
+            {% elif state.name in ['mixed', 'negative'] %}
+                class="icon-cross"
+            {%- endif -%}
+        ></span>
+    </div>
```

Comment:
In the meeting we said that we want the *Mixed* state to have a tilt icon. We'll need to add a new one in our `_icons.scss` file cause using the character `~` is not very consistent and needs special styling.
As for the *Conflict* state, we said to use a yellow color, which will be the same as *Mixed*. Is that rly what we want?